src/npc.cpp

```diff
@@ -1172,7 +1172,7 @@ void npc::make_angry()
         return; // We're already angry!
     }
 
-    add_msg( m_debug, "%s gets angry", name.c_str() );
+    add_msg( "%s gets angry", name.c_str() );
```

Comment:
This is still an issue. You need to enclose it in the translation macro `_( )` and add a sight check (`if( g->u.sees( *this )`). Or make it debug only again.